CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## [2.23.0] (Unreleased)
+
+BREAKING CHANGES:
+
+ - Remove `terraform.languageServer.maxNumberOfProblems` ([1062](https://github.com/hashicorp/vscode-terraform/pull/1062))
```

Comment:
I'd add some context here so it doesn't look as scary. We are unlikely to break anything since the setting was not used since `v2.0.0`.